src/components/Guide/Pointer.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
can we not get away with using CSS triangles? if we have to use SVGs, should that that svg in a separate file and fix webpack if need be

Reply:
or rather, maybe change the tooltip to take a render prop so that we can let the consumer do whatever they want to the tooltip container?

Reply:
I think this component on itself can remain the same, and I may change the `<Guide />` component in the future to accept a `pointer` render prop